### PR TITLE
adjust get started wait

### PR DESCRIPTION
**Pull request by chrisleewilcox**
**Development & PR Process**
1. Follow MetaMask [Mobile Coding Standards](https://docs.google.com/document/d/1VJLwTRsUw_5EDq_o8d6sSbXUAYENLurkRitYO45eY5o/edit?usp=sharing)
2. Add `release-xx` label to identify the PR slated for a upcoming release (will be used in release discussion)
3. Add `needs-dev-review` label when work is completed
4. Add `needs-qa` label when dev review is completed
5. Add `QA Passed` label when QA has signed off

**Description**
removed hard wait and use waitAndTap instead of tap
image

**Screenshots/Recordings**

_If applicable, add screenshots and/or recordings to visualize the before and after of your change_

**Issue**

Progresses #???

**Checklist**

* [ ] There is a related GitHub issue
* [ ] Tests are included if applicable
* [ ] Any added code is fully documented

### CODE REVIEW

File: wdio/step-definitions/common-steps.js [hunk 1]
```diff
@@ -14,7 +14,6 @@ import OnboardingWizardModal from '../screen-objects/Modals/OnboardingWizardModa
 Given(/^I have imported my wallet$/, async () => {
   const validAccount = Accounts.getValidAccount();
   await WelcomeScreen.isScreenTitleVisible();
```
Comment (SamuelSalas) [inline wdio/step-definitions/common-steps.js:16]: I see that WelcomeScreen.isScreenTitleVisible() method also has a hardcoded pause. is it possible to remove it?

**Review by cortisiko — APPROVED** [review]

File: wdio/step-definitions/common-steps.js (hunk 1 above)
Comment (chrisleewilcox) [reply wdio/step-definitions/common-steps.js:16]: When I remove the hard coded 20sec wait from .isScreenTitleVisible() tests are having issues running the next test and causing a lot of failures.  Not sure why.

Comment (chrisleewilcox) [reply wdio/step-definitions/common-steps.js:16]: Removed a 13sec pause and a 50sec pause.  Would probably need revisit and deep dive into the 20sec pause in .isScreenTitleVisible().

Comment (SamuelSalas) [reply wdio/step-definitions/common-steps.js:16]: Sounds fantastic

**Pull request merged**